src/foundry/client-esm/canvas/sources/point-light-source.d.mts

```diff
@@ -0,0 +1,46 @@
+import BaseLightSource from "./base-light-source.mts";
+import type { PointEffectSourceMixin_BaseLightSource_Interface } from "./point-effect-source.d.mts";
+import type PointEffectSourceMixin from "./point-effect-source.d.mts";
+
+declare const MixedPointBaseLightSource: PointEffectSourceMixin_BaseLightSource_Interface;
```

Comment:
For the record I only exported the interfaces in this `point-effect-source` and created the `consts` here so that they wouldn't show up in the runtime exports of any file.

Otherwise it'd technically be possible to import a variable that doesn't exist at runtime. It's not a _huge_ deal I just didn't want to leave it as even possible and I figured it would at least clutter up intellisense. There's nothing I can do about the interfaces though, at least if I want to stay DRY and not recreate it at every call site.